python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -96,52 +97,51 @@ def __call__(self, catalog):
 
 
 def plotText(textStr, fig, axis, xLoc, yLoc, prefix="", fontSize=None, color="k", coordSys="axes", **kwargs):
-    """Label the plot with the string provided at a given location
+    """Label the plot with the string provided at a given location.
 
     Parameters
     ----------
     textStr : `str`
-       String of text to plot.
+        String of text to plot.
     fig : `matplotlib.figure.Figure`
-       The figure to plot ``textStr`` on.
+        The figure to plot ``textStr`` on.
     axis : `matplotlib.axes._axes.Axes`
-       Particular matplotlib axes of ``plt`` on which to plot ``testStr``.
+        Particular matplotlib axes of ``plt`` on which to plot ``testStr``.
     xLoc, yLoc : `float`
-       x and y coordinates, in corrdinate system set by ``coordSys``, at which
-       to plot the ``textStr``.  The string will be centered both horizontally
-       and vertically at this position.
+        x and y coordinates, in corrdinate system set by ``coordSys``, at which
+        to plot the ``textStr``.  The string will be centered both horizontally
+        and vertically at this position.
     prefix : `str`, optional
-       Optional prefix to add to ``textStr``.
+        Optional prefix to add to ``textStr``.
     fontSize : `int` or `str` or `None`, optional
-       Size of font for plotting of ``textStr``.  May be either an absolute
-       font size in points, or a size string, relative to the default font
-       size.  Default is `None`, in which case an automatic scaling based
-       on the length of ``textStr`` will be used.
+        Size of font for plotting of ``textStr``.  May be either an absolute
+        font size in points, or a size string, relative to the default font
+        size, or `None`.  If `None`, in which case an automatic scaling based
+        on the length of ``textStr`` will be used.
     color : `str`, optional
-       Color to plot ``textStr``.  Can be any matplotlib color str.  Default is
-       k (for black).
+        Color to plot ``textStr``.  Can be any matplotlib color `str`.
     coordSys : `str`, optional
-       Coordinate system for ``xLoc``, ``yLoc``.  Choices and matplotlib
-       mappings are:
-       axes => axis.transAxes [the default]
-       data => axis.transData
-       figure => axis.transFigure
+        Coordinate system for ``xLoc``, ``yLoc``.  Choices and matplotlib
+        mappings are:
+            axes => axis.transAxes
+            data => axis.transData
+            figure => axis.transFigure
```

Comment:
I would put `"` around each left-hand side acceptable `str`. Maybe even double-backtick them?

I guess things on the right-hand side can probably be single-backticked as they're matplotlib objects.

Reply:
Now reads:
```
            "axes" => axis.transAxes (`matplotlib.transforms.BboxTransformTo`)                                 
            "data" => axis.transData (`matplotlib.transforms.BboxTransformTo`)                                 
            "figure" => axis.transFigure                                                                       
                        (`matplotlib.transforms.BboxTransformTo`)                                              

```